### ISSUE TITLE
Contrast and Color Accessibility

### ISSUE THREAD

**Issue by mariamatlick** [info442-sp21/c-team#50]
Description: It is very difficult to see the black text and brown buttons on a dark brown background. This renders your application inaccessibility for people to perceive the information, especially if you have colorblind or otherwise visually impaired users. 
Requested feature: Use developer tools or other accessibility tools to make sure that your buttons and headers have enough color contrast give your website legibility and accessibility.

![image](https://user-images.githubusercontent.com/49167660/119571670-7d36f680-bd66-11eb-877b-b5403ddf2e00.png)

**Comment by yejukim** [comment]
TO DO - The designed prototype of our application contains color contrasts, with white font against the brown background.

**Comment by cherimarie** [comment]
Great catch, Maria.

**Comment by yejukim** [comment]
We have changed it so the buttons and headers have more of color contrast in our latest release--it should have a white font against the brown background (v.1.2.0)
